project/blocky_game/player/avatar_interaction.gd

```diff
@@ -34,6 +34,8 @@ const _hotbar_keys = {
 @onready var _water_updater : WaterUpdater
 @onready var _terrain : VoxelTerrain = get_node("/root/Main/Game/VoxelTerrain")
 
+const _library := preload("res://blocky_game/blocks/voxel_library.tres")
```

Comment:
No need to load this from here, you already have access to it with `_block_types.get_model_library()`